android/src/main/java/com/plaid/PlaidModule.kt

```diff
@@ -76,6 +79,46 @@ class PlaidModule internal constructor(reactContext: ReactApplicationContext) :
     return builder.build()
   }
 
+  @ReactMethod
+  fun create(
+    token: String,
+    noLoadingState: Boolean,
+    logLevel: String,
+  ) {
+    val tokenConfiguration = getLinkTokenConfiguration(token, noLoadingState, getLogLevel(logLevel))
+    if (tokenConfiguration == null) {
+      throw LinkException("Unable to open link, please check that your configuration is valid")
+    }
+
+    // Set the event listener for Layer use cases.
```

Comment:
this isn't only for layer use cases right?

Reply:
Good point. We have to set it here for the layer events (otherwise we could set it in open)